loadtest/error.go

```diff
@@ -11,6 +20,10 @@ func IsInvalidConfig(err error) bool {
 	return microerror.Cause(err) == invalidConfigError
 }
 
+var invalidExecutionError = &microerror.Error{
```

Comment:
Can we do it like in https://github.com/giantswarm/aws-operator/pull/1796? Move to the top and add comment.

Reply:
I renamed the error to `failedLoadTestError` because I need to match on it. So this is actually dead code.

I missed this in the PRs yesterday so thx for the pointer. TIL #1